src/SonarScanner.MSBuild.PreProcessor/ProcessedArgs.cs

```diff
@@ -131,12 +131,7 @@ public string PropertiesFileName
             this.sonarServer = GetSonarServer(logger, isHostSet, sonarHostUrl, isSonarcloudSet, sonarcloudUrl);
```

Comment:
clean as you code: the default constructor is no longer used and can be removed

Reply:
It is used for mocking (via reflection)